tools/clang/test/CodeGenHLSL/RaceCond4.hlsl

```diff
@@ -0,0 +1,18 @@
+// RUN: %dxc -E main  -T cs_6_0 %s | FileCheck %s
+
+// CHECK: Race condition writing to shared memory detected, consider making this write conditional
```

Comment:
add `error: ` to make it more clear that we are checking for validation error here.